### ISSUE TITLE
Add support for non-crate dependencies

### ISSUE THREAD

**Issue by gyakovlev** [gentoo/cargo-ebuild#5]
Sometimes a project uses a git dependency.
cargo-ebuild should handle simple cases of git deps.

**Comment by leonardohn** [comment]
The main changes required for this to work should be on cargo.eclass. There should be a variable similar to `CRATES` to allow the maintainer to add vendored dependency paths relative to the workdir. It could be done by generating (or modifying) the `.cargo/config` file so cargo can handle those dependencies.

**Comment by thesamesam** [comment]
I suppose this covers the case I mentioned in https://github.com/projg2/pycargoebuild/issues/14#issue-1557734676 wrt pkgcraft-c's '../pkgcraft' dep too?

**Comment by gyakovlev** [comment]
Not really, that’s different. It’s already working fine if you invoke cargo(eclass included) properly.
